### ISSUE TITLE
Dragging Swimlanes still does not work with 28 lanes

### ISSUE THREAD

**Issue by doniels** [wekan/wekan#4219]
## Issue

We cannot drag swimlanes on the board with 28 swimlanes. Not new swimlanes and not old ones. 

Basically, https://giters.com/wekan/wekan/issues/4098 still happens by us - Using  Windows with Chrome Version 96.0.4664.45 .

On the smaller board with only 12 swimlanes we can move swimlanes.

We have the latest version of WeKan
![image](https://user-images.githubusercontent.com/57865526/145015812-884b7984-4772-48b8-ab6a-2bb6e7582554.png)

**Comment by xet7** [comment]
@doniels 

Please check is your server swapping, with:
```
free -h
```
If it is, please increase amount of total RAM to 4 GB.

Then please add comment about did it help.

**Comment by xet7** [comment]
@doniels 

Please also check that there is enough free disk space:
```
df -h
```

**Comment by doniels** [comment]
@xet7 - thanks for the reply. Here are the results.

> free -h

We were at 2Gb - we upgraded to 4Gb.

> df -h

```
              total        used        free      shared  buff/cache   available
Mem:           1.9G        907M        202M        1.3M        845M        871M
Swap:            0B          0B          0B
```

Results: Still cannot drag the 28th swimlane to the left.

**But** - I managed to drag swimlane #7 to the extreme right - and then drag  the new swimlane to its correct location (#8) - and them move then 7th swimlane back to its correct location (#7).

I did not try this before we increased the RAM.

So, something is still broken - with 28 swimlanes the dragging is not working properly.
